lib/modules/schedule/widget/home_widgets/clocking_widget.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:ui_collection/widget/colors/common_color.dart';
+
+class ClockingWidget extends StatelessWidget {
+  final String title;
+  final IconData icon;
+  const ClockingWidget({Key? key, required this.title, required this.icon})
+      : super(key: key);
+
+  ButtonStyle getButtonStyle(String title) {
```

Comment:
make this function private by putting a `_` prefix in the function name example: `_getButtonStyle`

Reply:
Move these functions after `build` method